chipsec/helper/file/helper.py

```diff
@@ -0,0 +1,297 @@
+#!/usr/bin/python
+#CHIPSEC: Platform Security Assessment Framework
+#Copyright (c) 2018, Intel Corporation
+# 
+#This program is free software; you can redistribute it and/or
+#modify it under the terms of the GNU General Public License
+#as published by the Free Software Foundation; Version 2.
+#
+#This program is distributed in the hope that it will be useful,
+#but WITHOUT ANY WARRANTY; without even the implied warranty of
+#MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#GNU General Public License for more details.
+#
+#You should have received a copy of the GNU General Public License
+#along with this program; if not, write to the Free Software
+#Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+#Contact information:
+#chipsec@intel.com
+#
+
+"""
+Use results from a json file
+"""
+
+import chipsec.file
+from chipsec.logger import *
+from chipsec.helper.oshelper import Helper
+import json
+
+class FileCmds:
+    def __init__(self, filename):
+        self.data = {}
+        if filename == "":
+            self.filename = "clone.json"
+        else:
+            self.filename = filename
+
+    def AddElement(self,cmd,args,ret):
```

Comment:
Can we do something like push/pop cmd/args/ret so that when we read/write the same address multiple times, we save the various values in order? I don't think this should be blocking acceptance, but it would probably avoid problems later to have it.

Reply:
Added a list to AddElement, which should resolve the concern.  Note that insert() was used opposed to append() to get first in first out when using pop() within getElement. Let me know if other changes are necessary.